### DISCUSSION TITLE
Suggestions for new drone exercises

### DISCUSSION THREAD

**Discussion by pariaspe** [JdeRobot/RoboticsAcademy#1247] [Ideas]
We are exploring the creation of new drone exercises.
Some current ideas are:
- Drone visual inspection over high voltage towers.
- Drone package delivery.

Any further suggestion is welcome!

**Comment by RUFFY-369** [comment]
Hi @pariaspe , how about Drone racing or may be just with even one drone to beat a time record.
And also can we incorporate multiple cats in the drone cat and mouse exercise to chase a faster mouse?

**Reply by jmplaza** [reply]
We already have several mice for the drone-cat-mouse exercise, slow one, a midium one and a fast one. Feel free to test them :-)

Regarding drone racing, we could see the ["drone gymkhana" exercise ](https://jderobot.github.io/RoboticsAcademy/exercises/Drones/drone_gymkhana) as that. 

In general automatic assessment (such as time records) is not going to be added to RoboticsAcademy but to its cousin open platform Unibotics.

**Reply by RUFFY-369** [reply]
No, I was thinking of having multiple cats to chase one mouse with avoiding collisions.

Yeah drone gymkhana can be considered as one.

Oh! Alright

**Reply by jmplaza** [reply]
For introducing the avoid collisions requirement in the cat-drone programming maybe we could include static objects instead of additional drones, as the more robots in the Gazebo world, the more computing power is required to run fluently the scenario.

**Reply by RUFFY-369** [reply]
Yeah that will become the bottleneck otherwise, we can do  on-the-fly selection of Gazebo world as you said for static objs as well.
